### PR TITLE
Redesign 404 page for Clientside (#319)

### PR DESCRIPTION
**Pull request by Ram81**
#319 resolved. @aka-jain please review

### CODE REVIEW

File: frontend/src/js/directives/directives.js [hunk 1]
```diff
@@ -288,5 +288,4 @@
             var slideCompObj = new slideComp();
         }
     }
-
-})();
+})();
```
Comment (aka-jain) [inline frontend/src/js/directives/directives.js]: @Ram81 : please give a new line here.

File: tests/unit/evalai/test_views.py [hunk 2]
```diff
@@ -14,4 +14,4 @@ def test_error_handlers_404(self):
         request = c.get("/abc")
         response = page_not_found(request)
         self.assertEqual(request.status_code, response.status_code)
-        self.assertIn('Error 404', unicode(response))
+        self.assertIn('404', unicode(response))
```
Comment (aka-jain) [inline tests/unit/evalai/test_views.py]: @deshraj : Is it neccessary to do this?

File: frontend/src/js/directives/directives.js (hunk 1 above)
Comment (Ram81) [inline frontend/src/js/directives/directives.js]: @aka-jain  Done.

File: frontend/src/js/directives/directives.js [hunk 3]
```diff
@@ -288,5 +288,5 @@
             var slideCompObj = new slideComp();
         }
     }
-
-})();
+    
+})();
```
Comment (deshraj) [inline frontend/src/js/directives/directives.js]: Please add a newline here.

File: tests/unit/evalai/test_views.py (hunk 2 above)
Comment (deshraj) [inline tests/unit/evalai/test_views.py]: NO, @Ram81 can you please revert it back.

**Comment by aka-jain** [comment]
@Ram81 : Your PR looks good, but do the required changes that have mentioned in the comments and let's merge this :)

**Comment by Ram81** [comment]
@aka-jain  If I remove the changes of test_view.py the checks are failing, please suggest something?

**Comment by aka-jain** [comment]
@deshraj : help please!

**Comment by Ram81** [comment]
@aka-jain  I think the issue is with the test_views.py ?

File: frontend/src/js/directives/directives.js (hunk 3 above)
Comment (Ram81) [inline frontend/src/js/directives/directives.js]: @deshraj  I did add one do you want me to add one more?

**Comment by aka-jain** [comment]
Yup, let @deshraj to handle this.

**Comment by Ram81** [comment]
:+1:

File: frontend/src/views/web/error-pages/error-404.html [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
+	  	<div class="fof-stars" id="star-6"></div>
+	  	<div class="fof-stars" id="star-7"></div>
+	  	<div class="fof-stars" id="star-8"></div>
+	  	<div class="fof-stars" id="star-9"></div>
+	  	<div class="fof-stars" id="star-10"></div>
+	  	<div class="fof-stars" id="star-11"></div>
+	  	<div class="fof-stars" id="star-12"></div>
+	  	<div class="owl-pic"></div>
+    	<div class="fof-error">
+    		404
```
Comment (deshraj) [inline frontend/src/views/web/error-pages/error-404.html]: @Ram81 all the checks will pass if you change it to `Error 404` or change the `test_views.py` that you changed earlier.

Comment (Ram81) [inline frontend/src/views/web/error-pages/error-404.html]: @deshraj  I will do the change in test_views.py as I did previously!

Comment (deshraj) [inline frontend/src/views/web/error-pages/error-404.html]: Cool....sounds good. :-)

**Comment by Ram81** [comment]
@aka-jain  @deshraj  please review

File: frontend/src/css/modules/error_404.scss [hunk 5]
```diff
@@ -4,28 +4,203 @@
...
+	background: #ffffff;
+	margin-top: 100px;
+}
+
+.fof-grad {
+	margin: 0 auto;
+	position: relative;
+	overflow: hidden;
+	height: 450px;
+    width: 450px;
```
Comment (aka-jain) [inline frontend/src/css/modules/error_404.scss]: @Ram81 : Last change, please make code properly indented

**Comment by Ram81** [comment]
Sorry for this will correct and push back

**Comment by Ram81** [comment]
@aka-jain  Please review!

**Comment by aka-jain** [comment]
Great work @Ram81, keep it up :)
@deshraj: Merge it please!

**Comment by Ram81** [comment]
Thanks @aka-jain

**Pull request merged**